x-pack/test/alerting_api_integration/spaces_only/tests/index.ts

```diff
@@ -8,28 +8,26 @@ import { FtrProviderContext } from '../../common/ftr_provider_context';
 import { Spaces } from '../scenarios';
 
 // eslint-disable-next-line import/no-default-export
-export default function alertingApiIntegrationTests({
-  loadTestFile,
-  getService,
-}: FtrProviderContext) {
-  const spacesService = getService('spaces');
-  const esArchiver = getService('esArchiver');
-
+export default function alertingApiIntegrationTests({ loadTestFile }: FtrProviderContext) {
   describe('alerting api integration spaces only', function () {
     this.tags('ciGroup9');
 
-    before(async () => {
-      for (const space of Object.values(Spaces)) {
-        if (space.id === 'default') continue;
-
-        const { id, name, disabledFeatures } = space;
-        await spacesService.create({ id, name, disabledFeatures });
-      }
-    });
-
-    after(async () => await esArchiver.unload('empty_kibana'));
-
     loadTestFile(require.resolve('./actions'));
     loadTestFile(require.resolve('./alerting'));
   });
 }
+
+export async function buildUp(getService: FtrProviderContext['getService']) {
+  const spacesService = getService('spaces');
+  for (const space of Object.values(Spaces)) {
+    if (space.id === 'default') continue;
+
+    const { id, name, disabledFeatures } = space;
+    await spacesService.create({ id, name, disabledFeatures });
+  }
+}
+
+export async function tearDown(getService: FtrProviderContext['getService']) {
+  const esArchiver = getService('esArchiver');
+  await esArchiver.unload('empty_kibana');
+}
```

Comment:
I'm confused... why is this here?
Didn't we merge this into Main already? 😮
We might need a fresh merge from Main into the PR... 🤔

Reply:
oh dang, was going to comment in the PR on this.

There was a merge conflict w/master in this area, and rather than merge master, I just copied the code in here.    I was still rebasing at the time, didn't want to have to merge master, but forgot to add a note about it.

I did actually have to merge master this morning and guess what!  It's gone!